kuma/api/v1/views.py

```diff
@@ -37,7 +37,10 @@
 )
 from kuma.search.search import SearchView
 from kuma.users.models import User, UserSubscription
-from kuma.users.stripe_utils import create_stripe_customer_and_subscription_for_user
+from kuma.users.stripe_utils import (
+    create_stripe_customer_and_subscription_for_user,
+    retrieve_and_synchronize_subscription_info,
```

Comment:
This is the big deal. `kuma/users/stripe_utils.py` is now the new central home for these big convenience functions. They get used from `kuma/users/views.py` and `kuma/payments/views.py` and `kuma/api/v1/views.py`. (plus some other things like `users/signal_handlers.py`)

It's an unfinished book what to do exactly with these things and where the ideal place is. I actually thing `kuma.users` is **the wrong place** but I think it all kinda started (a couple of months ago) because we added the Subscriptions 2020 to Edit Profile initially. 

Perhaps when this last batch of re-arrangements is done, we go in and move `stripe_utils.py` from `kuma/users/` to `kuma/payments/`. That way, `kuma/users` gets smaller and `kuma/payments/` can be the home for all things Stripe related.